### PR TITLE
More MVVM for Contributors view

### PR DESCRIPTION
**Pull request by k-kagurazaka**
## Issue
None

## Overview (Required)
The purpose of this PR is `ContributorsFragment` makes thin and `ContributorsViewModel` makes testable.

- Apply MVVM pattern to Toolbar title.
- Move logic (rendering contributors) from `ContributorsFragment` to `ContributorsViewModel`.
- Add tests for `ContributorsViewModel`.

### CODE REVIEW

**Comment by konifar** [comment]
Cool 🆒 Thanks for interesting suggestion!
I think it's a little complicated and too much for me 🤔 
Please let me consider 🙇

File: app/src/main/java/io/github/droidkaigi/confsched2017/di/AppModule.java [hunk 1]
```diff
@@ -104,6 +106,21 @@ public GoogleFormService provideGoogleFormService(OkHttpClient client) {
                 .create(GoogleFormService.class);
     }
 
+    @Provides
+    public ResourceResolver provideResourceResolver(final Context context) {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/di/AppModule.java]: Cool 🆒 I love this idea 😃

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java [hunk 2]
```diff
@@ -16,14 +16,29 @@
...
+import timber.log.Timber;
 
+@FragmentScope
 public final class ContributorsViewModel extends BaseObservable implements ViewModel, ContributorViewModel.Callback {
 
+    public static final String TAG = ContributorsViewModel.class.getSimpleName();
+
+    private final ResourceResolver resourceResolver;
+
+    private final ToolbarViewModel toolbarViewModel;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java]: I know what you want to do.
But nested ViewModel makes me feel a little complicated.
Is it common way for MVVM? 🤔 Sorry for my poor knowledge 🙇

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java]: But `ObservableList<ContributorViewModel> viewModels` has already been nested.
So I feed it's not big problem. It's just question!

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ToolbarViewModel.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import com.android.databinding.library.baseAdapters.BR;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+
+import javax.inject.Inject;
+
+import io.github.droidkaigi.confsched2017.di.scope.ActivityScope;
+
+@ActivityScope
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ToolbarViewModel.java]: 👍

**Comment by konifar** [comment]
> I think it's a little complicated and too much for me 🤔
> Please let me consider 🙇

Finally I'm for your suggestion. It's awesome. As you said, this PR makes easy to test 👍 
I learned a lot from this!

**Comment by konifar** [comment]
LGTM! 👍 
But sorry, conflict occurred. Can you merge master?

**Comment by k-kagurazaka** [comment]
I resolved conflicts 👍

**Comment by konifar** [comment]
Thank you for corresponding till late at night (morning?) 😄

**Comment by k-kagurazaka** [comment]
No problem.
Thank YOU!

**Comment by konifar** [comment]
Here we go!

**Pull request merged**